Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	pipenv install
+	pipenv install --dev
+
+test:
+	pipenv check --style ./response_operations_ui ./tests
+	pipenv run python run_tests.py
```

Comment:
Should we have a make command to run the application as well?

Reply:
Also if we've got a makefile now we should probably update the readme to mention the new commands